### PR TITLE
Removing unused unforgeable name predicting tools and api (1st part)

### LINKED ISSUE #3690: Removing unused unforgeable name predicting tools and api
## Overview

After #3687 , the user who want to use `InsertSigned` would not need an explicit signature in the rholang term anymore.

### Impact

Users who is using `/api/prepare-deploy` should change their codes.

### Design

1. https://github.com/rchain/rchain/blob/4f94bf706a6943c055b95e65609f8cc3073eea33/casper/src/main/scala/coop/rchain/casper/rholang/RegistrySigGen.scala
This file could be totally removed because it is used to generate the signature of the `InsertSigned`. The generated contents are in https://github.com/rchain/rchain/blob/4f94bf706a6943c055b95e65609f8cc3073eea33/casper/src/main/resources/AuthKey.rho#L1-L16

2. :heavy_check_mark: The `/api/prepare-deploy` is not longer userful once [we change how random seed comes from](https://github.com/rchain-test/rchain-secure/issues/3). We should remove this api and the related codes. https://github.com/rchain/rchain/blob/4f94bf706a6943c055b95e65609f8cc3073eea33/node/src/main/scala/coop/rchain/node/api/WebApi.scala#L74-L88 should be removed and https://github.com/rchain/rchain/blob/4f94bf706a6943c055b95e65609f8cc3073eea33/casper/src/main/scala/coop/rchain/casper/api/BlockApiImpl.scala#L515-L524 should be removed

3. :heavy_check_mark: remove the related grpc definitions https://github.com/rchain/rchain/blob/2399df3fe7a996bb53de2c9174b6f5feae73b167/models/src/main/protobuf/DeployServiceV1.proto#L53-L54 and https://github.com/rchain/rchain/blob/2399df3fe7a996bb53de2c9174b6f5feae73b167/models/src/main/protobuf/DeployServiceCommon.proto#L74-L78 and the related api https://github.com/rchain/rchain/blob/2399df3fe7a996bb53de2c9174b6f5feae73b167/node/src/main/scala/coop/rchain/node/api/DeployGrpcServiceV1.scala#L179-L189

### Resource estimation

One day

### PR DESCRIPTION
**Pull request by maximus19910112**
## Overview
1st part of the [issue](https://github.com/rchain/rchain/issues/3690)

### Notes
1. Removed file https://github.com/rchain/rchain/blob/4f94bf706a6943c055b95e65609f8cc3073eea33/casper/src/main/scala/coop/rchain/casper/rholang/RegistrySigGen.scala
2. Adaptation of code

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Review by tgrospic — APPROVED** [review]
LGTM

File: casper/src/test/scala/coop/rchain/casper/addblock/MultiParentCasperAddBlockSpec.scala [hunk 1]
```diff
@@ -23,8 +21,8 @@ import coop.rchain.shared.scalatestcontrib._
 import coop.rchain.shared.syntax._
 import monix.eval.Task
 import monix.execution.Scheduler.Implicits.global
-import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.Inspectors
+import org.scalatest.flatspec.AnyFlatSpec
```
Comment (stanislavlyalin) [inline casper/src/test/scala/coop/rchain/casper/addblock/MultiParentCasperAddBlockSpec.scala]: Why this imports reordering?

**Review by stanislavlyalin — APPROVED** [review]
LGTM

File: casper/src/test/scala/coop/rchain/casper/addblock/MultiParentCasperAddBlockSpec.scala (hunk 1 above)
Comment (maximus19910112) [reply casper/src/test/scala/coop/rchain/casper/addblock/MultiParentCasperAddBlockSpec.scala]: I used command Code -> Optimize Imports to delete redundant imports. Maybe this operation changed order of imports. Now I restored initial sequence

**Pull request merged**